index.html

```diff
@@ -15,43 +15,46 @@
+          <div class="logo">
+            <span>My Logo</span>
+          </div>
+          <nav>
+            <ul class="menu">
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+            </ul>
+          </nav>
+        </div>
+      </div>
     </header>
     <main>
-    <section class="headline">
-          <h1 class="primary-text-2">I'm Zermarden <br>
-          Glad to see <br>
-          you!</h1>
-          <p class="primary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-          <div class="social-media">
-            <div class="supporting-text">
-              <span>LET'S CONNECT!</span>
-            </div>
-          </div>
-          <div class="social-media-icons">
-            <ul class="social-media-icons-list">
-              <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
-              <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
-              <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="16"></a></li>
-              <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="18" height="20"></a></li>
-              <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="15" height="20"></a></li>
-          </ul>
-          </div>
-        </section>
-        <!--Images should be in Headline section. After that check CSS-->
+      <section class="headline">
+        <h1 class="hello">I'm Zermarden<br>
+          Glad to see you!</h1>
+        <p class="hello-text">I’m a software developer! I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a project you need 
+            coded, don’t hesitate to contact me. </p>
+        <h4 class="let-s-connect">LET'S CONNECT!</h4>
+        <ul class="social-list">
+          <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="16"></a></li>
+          <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="18" height="20"></a></li>
+          <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="15" height="20"></a></li>
+        </ul>
         <img class="header-shapes" src="assets/bg.png" alt="header-shapes">
-        <img class="header-background" src="assets/header-background-mobile.svg" alt="background-image">
-        <div class="works"> <!--Change it to section-->
+      </section>
+      <section class="works" id="portfolio">
```

Comment:
- Please in the work section make use of the proper button interaction for the `See projects` 
##### Figma Design: Button interaction
![Screenshot from 2022-06-01 19-12-14](https://user-images.githubusercontent.com/58771507/171915913-ba9c9d9e-72fe-4df8-9501-61af1001b7d5.png)

##### Your Design  Button interaction

https://user-images.githubusercontent.com/58771507/171916048-98ea6e42-e8d6-448f-9d50-eba6a93dd136.mp4